src/utils/license.js

```diff
@@ -31,9 +31,8 @@ export const getFullLicenseName = (license, license_version) => {
 /**
  * CC licenses have different legal status from the public domain marks
  * such as CC0 and PDM, and need different wording.
- * Returns false if licenseOrMark is CC0 or PDM
  * @param {License|LicenseName|''} licenseOrMark
- * @return {boolean}
+ * @return {boolean} false if licenseOrMark is CC0 or PDM, true otherwise
```

Comment:
Funny, I think we can actually do better with this return type, if you're interested :slightly_smiling_face: 

```suggestion
 * @return {asserts licenseOrMark is Exclude<'pdm' | 'cc0', License | LicenseName | ''>} false if licenseOrMark is CC0 or PDM, true otherwise
```

Reply:
It's as if return types were their own little language!